core/templates/dev/head/pages/load_constants.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var request = new XMLHttpRequest();
+request.overrideMimeType('application/json');
+request.open('GET', '/assets/constants.json', false);
+request.send(null);
+var CONSTANTS = JSON.parse(request.responseText);
```

Comment:
I don't have a strong preference for individual constants vs a CONSTANTS object. However, in the case where we have just one object, I think copying the whole object each time would be wasteful. 

Perhaps another option would be to use a provider to retrieve individual constants as needed, similar to how Angular's template provider retrieves individual templates.